packages/expo-video-thumbnails/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### 💡 Others
 
+## 6.2.0 — 2021-12-14
+
+- Thumbnails can now be generated with `content://` paths on Android. ([#15553](https://github.com/expo/expo/pull/15553) by [@lukebrandonfarrell](https://github.com/lukebrandonfarrell))
```

Comment:
Sorry, I didn't notice it before 🤦‍♂️ Can you move the changelog entry a bit higher, to the "Unpublished/New Features" section? `6.2.0` is not published yet and can't say when it will be 😄

Reply:
Done :)